src/seedu/addressbook/AddressBook.java

```diff
@@ -643,66 +678,71 @@ private static String getDisplayString(ArrayList<String[]> persons) {
         for (int i = 0; i < persons.size(); i++) {
             final String[] person = persons.get(i);
             final int displayIndex = i + DISPLAYED_INDEX_OFFSET;
-            messageAccumulator.append('\t')
-                              .append(getIndexedPersonListElementMessage(displayIndex, person))
-                              .append(LS);
+            messageAccumulator.append('\t').append(getIndexedPersonListElementMessage(displayIndex, person)).append(LS);
```

Comment:
Is this way of indenting more readable or was the previous indentation better?
Or is this caused when using eclipse's auto indetation?